engine/app/components/citizens_advice_components/textarea.html.haml

```diff
@@ -1,2 +1,3 @@
 = render CitizensAdviceComponents::Input.new(base_input_args) do
   %textarea.cads-textarea{ input_attributes }
+    = value
```

Comment:
Do you want to validate that the value is non-blank? Because in the `set_options` method it could be fetched as a nil value, but then I guess that's actually not a problem for an auto-entered value right 🤷

Reply:
Yeah it's fine if it's nil, it just renders nothing which is correct